README.md

```diff
@@ -1,108 +1,190 @@
+export $(cat .env | grep -v "#" | xargs)
+```
+
+## 4. Memory Management Settings
+
+> For additional memory management information see the Memory configuration notes section
+
 As OpenCTI has a dependency on ElasticSearch, you have to set the `vm.max_map_count` before running the containers, as mentioned in the [ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode).
 
 ```bash
 $ sudo sysctl -w vm.max_map_count=1048575
-
 ```
 
-To make this parameter persistent, please update your `/etc/sysctl.conf` file and add the following line at the end:
+To make this parameter persistent, add the following to the end of your `/etc/sysctl.conf`:
 
 ```bash
 $ vm.max_map_count=1048575
 ```
 
-## Run
-
-### Using single node Docker
+## 5. Run OpenCTI - Full-stack, including UI
 
-You can deploy without using Docker swarm, with a the `docker-compose` command. After changing your `.env` file, just type:
+### Single node Docker - *not* Docker swarm
 
+After changing your `.env` file run `docker-compose` in detached (`-d`) mode:
 ```bash
 $ sudo docker-compose up -d
 ```
 
-### Using Docker swarm
+### Docker swarm
 
-In order to have the best experience with Docker, we recommend using the Docker stack feature. In this mode you will have the capacity to easily scale your deployment. If your virtual machine is not a part of a Swarm cluster, please use:
+In order to have the best experience with Docker, we recommend using the Docker stack feature. In this mode you will have the capacity to easily scale your deployment. 
+
+If your virtual machine is not already part of a Swarm cluster, initialize a swarm:
 
 ```bash
 $ sudo docker swarm init
 ```
 
-Then, you have to put your environment variables in the `/etc/environment` and then:
+Put your environment variables in the `/etc/environment`:
 
 ```bash
+# If you already exported your variables to .env from above:
+$ sudo cat .env >> /etc/environment
 $ sudo source /etc/environment
 $ sudo docker stack deploy --compose-file docker-compose.yml opencti
 ```
 
 You can now go to [http://localhost:8080](http://localhost:8080/) and log in with the credentials configured in your environment variables.
 
-## Update
+## 6. Run OpenCTI infrastructure with UI/GraphQL in development mode
+
+In order to develop OpenCTI UI/GraphQL in the most efficient manner we have provided a `docker-compose.dev.yml` which stands up the back-end/infrastructure of OpenCTI, with the expectation that you will run the OpenCTI front-end (React/GraphQL) separately.
+
+This docker-compose exposes all necessary ports for the UI/GraphQL to attach to in order to support local development.
+
+To run the services required for local development run:
+```bash
+$ sudo docker-compose up -f docker-compose.dev.yml -d
+```
+
+To configure/run the UI/GraphQL we would direct you to the [Notion documentation](https://www.notion.so/Frontend-1278fff370304cf09f6fd54ffb06f0b4)
+
+# Appendices
```

Comment:
Created this section, since this is not strictly how to RUN OpenCTI, but rather additional actions to be performed as the user sees fit.